### ISSUE TITLE
can you share your complement version, there have many func not be done.   :D

### ISSUE THREAD

**Issue by AnyWayFree** [Jungle-Sven/avellaneda_stoikov_mm#1]
😊

**Comment by Jungle-Sven** [comment]
Yeah, i'll think about it. You should know that there is no single right way calculating volatility, analyzing orderbooks, etc. Avellaneda & Stoikov gave us only basic guidelines about their approach to modern market making in general, without details.

**Comment by AnyWayFree** [comment]
You are right. 
Maybe you can learn from HummingBot.
And your code is the most detailed I have seen. Have you backtested AS model?
